### PR TITLE
amara

### PR DESCRIPTION
**Pull request by hellodian**
Amara Defi

### CODE REVIEW

File: defi/amaralend.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+**Step 2：**Get test coins on Amara Finance Discord. The Amara team developed a faucet robot for your convenience. You will get test coins of BTC/USDT/DOT/KSM and gas fee (about 0.2 DEV) after addresses submitted.       
+
+**Step 3:** Deposit any one of the four assets (BTC/USDT/DOT/KSM).
+
+![step3](../images/amaralend/dapps-amaralend-2.png)
+
+**Step 4** “My assets” - “My deposit” - “Collateral”, switch on “use as collateral ”.
+
+![step4](/Users/hors.wei/moonbeam-project-directory/images/amaralend/dapps-amaralend-3.png)
```
Comment (albertov19) [inline defi/amaralend.md:83]: Should be:

```
![step4](../images/amaralend/dapps-amaralend-3.png)
```

File: defi/amaralend.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+**Step 5** Lend any one of the four assets and repay it.
+
+![step5](/Users/hors.wei/moonbeam-project-directory/images/amaralend/dapps-amaralend-4.png)
```
Comment (albertov19) [inline defi/amaralend.md:87]: Should be:

```
![step5](../images/amaralend/dapps-amaralend-4.png)
```

File: defi/amaralend.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+   
+
+## Moonbase Alpha Implementation
+
+AmaraLend Beta has been deployed on Moonbase Alpha, the testnet of Moonbeam, allowing depositing and lending of BTC, USDT, DOT, KSM. We are looking forward to having you with us. Access [here](https://lendtest.amara.link/)
+
+You can find a detailed tutorial on how to use AmaraLend beta: 
+
+https://amara.gitbook.io/amara-finance/
```
Comment (albertov19) [inline defi/amaralend.md:52]: Suggest:

```
[https://amara.gitbook.io/amara-finance/](https://amara.gitbook.io/amara-finance/)

**Review by albertov19 — CHANGES_REQUESTED** [review]
Thanks for your input. I've left some comments.

Also please, in the ./defi/.pages file you need to add `amaralend.md`, so:

```
title: DeFi
nav:
  - index.md
  - sushiswap.md
  - amaralend.md
  ```

File: defi/amaralend.md (hunk 3 above)
Comment (weiht12) [reply defi/amaralend.md:52]: thu suggestions，we will improve the documentation

**Review by albertov19 — CHANGES_REQUESTED** [review]
As a last comment, please add a logo image to: https://github.com/PureStake/moonbeam-docs/tree/master/images/index-pages/learn/dapps-list/defi

Similar to `sushiswap.png` please name it `amaralend.png` - It should be Square, follow the sushiswap image as example

Thanks!

**Comment by weiht12** [comment]
@albertov19 we have add amara logo to https://github.com/PureStake/moonbeam-docs/tree/master/images/index-pages/learn/dapps-list/defi，and we have named it amaralend.png.
Plz check it，It was a pleasure communicating with you and best wishes.

**Comment by hyd628** [comment]
Still seems to be missing the logo. @weiht12

**Comment by weiht12** [comment]
嗯，我这边会重新提交

Henry (Wei) ***@***.***> 于2021年9月27日周一 下午7:02写道：

**Comment by hyd628** [comment]
lgtm.

**Pull request merged**